onecodex/lib/upload.py

```diff
@@ -420,7 +423,7 @@ def _call_init_upload(file_name, file_size, metadata, tags, project, samples_res
     return upload_info
 
 
-def upload_sequence(files, session, samples_resource, threads=1, metadata=None, tags=None,
+def upload_sequence(files, session, samples_resource, threads=None, metadata=None, tags=None,
```

Comment:
Why not do `threads=DEFAULT_THREADS` here?

Reply:
I don't think that's necessary since `None` will fall through to `DEFAULT_THREADS` in lib/upload.py